lib/check_params.h

```diff
@@ -855,6 +855,12 @@ void printQudaMultigridParam(QudaMultigridParam *param) {
   P(is_staggered, QUDA_BOOLEAN_INVALID);
 #endif
 
+#ifdef INIT_PARAM
+  P(use_mma, QUDA_BOOLEAN_TRUE);
```

Comment:
We should make the default `false` for now because I know the staggered-specific memory swap headaches on the horizon. We can change it to `true` once those are in.

Reply:
Nevermind, leave as true. As discussed on slack add a hard-coded set to false in the MILC interface around here: https://github.com/lattice/quda/blob/feature/mg-mma/lib/milc_interface.cpp#L1693

Justification here is MILC HISQ MG is essentially always run in the deep oversubscribed regime, and the extra allocations could nuke performance without some `prefetch` games.

Reply:
Added.